app/views/layouts/application.haml

```diff
@@ -124,8 +124,8 @@
             %ul
               - flash[:error][:array].each do |error|
                 %li= error
-        - flash.delete(:error)
       - flash.keys.each do |key|
+        - next if key == "error" && flash[key].is_a?(Hash)
```

Comment:
What's up with this change here? Do we use `flash['error']` sometimes…?

Reply:
Oh, it's to avoid breaking `flash[:error]` in tests with `render_views`! Neat!

Reply:
That was breaking one of the galleries tests after I changed those to use a hash on failure!

Reply:
Sorted out with #1103!